README.md

```diff
@@ -10,10 +10,16 @@ Both `JUMP-Target-ORF` and `JUMP-Target-CRISPR` have sgRNAs and ORFs correspondi
 
 Each of the 3 lists fit on a single 384-well plate; the suggested plate layouts are provided (below).
 
-The target annotations were obtained from https://clue.io/repurposing. 
+The target annotations were obtained from https://clue.io/repurposing.
 
 This resource was created through the [JUMP-Cell Painting Consortium](https://jump-cellpainting.broadinstitute.org/).
 
+**Versioning**
+- There's a single version for the `JUMP-Target-CRISPR` and `JUMP-Target-CRISPR` plates  (`JUMP-Target-1-CRISPR` and `JUMP-Target-1-CRISPR` respectively)
+- There are two versions os `JUMP-Target-Compound` (`JUMP-Target-Compound-1` and `JUMP-Target-Compound-2`). Both have the same set of compounds, but have different Broad sample IDs and different layouts.
+
```

Comment:
@AnneCarpenter What should we say abo

The `JUMP-Target-Compound-2` layout is used in the production of the [JUMP dataset](https://jump-cellpainting.broadinstitute.org/results). We recommend using this layout because it will remove layout as a potential confounder when matching against the JUMP dataset.

Reply:
Yep!